configure.ac

```diff
@@ -307,20 +306,31 @@ case $set_ecmult_window in
 esac
 
 # Set ecmult gen precision
-if test x"$req_ecmult_gen_precision" = x"auto"; then
-  set_ecmult_gen_precision=4
+if test x"$req_ecmult_gen_kb" = x"auto"; then
+  set_ecmult_gen_kb=86
```

Comment:
I'm a little bit of the mind that 22 should be the default, but perhaps that's my old brain stuck in 64kB L1 world. Still, would it be reasonable to default to 22 for 32bit?

Reply:
With https://github.com/bitcoin-core/secp256k1/issues/929, it may be good idea to handle the logic in the preprocessor in the case of "auto". 

If you agree, I think it's both ok to do it in this PR, or postpone it to a separate PR that also cares about all other setting/parameters (which we should do anyway).

Reply:
I'm not convinced about what the defaults should be, but given that ecmult already defaults to 1 MiB precomputed table, I see little harm in picking an 86 KiB one here.

I agree we can easily revisit defaults later (especially in combination with #929).

Reply:
I changed the configure-driver default to also be 22 kB; we can revisit the defaults later.